pkg/acsengine/engine.go

```diff
@@ -570,7 +572,20 @@ func getParameters(cs *api.ContainerService, isClassicMode bool, generatorCode s
 			if dashboardAddon.Containers[c].Image != "" {
 				addValue(parametersMap, "kubernetesDashboardSpec", dashboardAddon.Containers[c].Image)
 			} else {
-				addValue(parametersMap, "kubernetesDashboardSpec", cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase+KubeConfigs[k8sVersion]["dashboard"])
+				addValue(parametersMap, "kubernetesDashboardSpec", cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase+KubeConfigs[k8sVersion][DefaultDashboardAddonName])
```

Comment:
@jackfrancis is there a reason why this wasn't using the name var? is it okay that I changed it?

Reply:
👍 and thanks!